src/Views/Auth/Auth.js

```diff
@@ -0,0 +1,57 @@
+import './Auth.css';
+import Form from '../../components/Form/Form.js';
+import Header from '../../components/Header/Header.js';
+import { useState } from 'react';
+import { logout } from '../../services/users.js';
+import React from 'react';
+import { signInUser, signUpUser } from '../../services/users';
+
+export default function Auth({ user, setUser }) {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [message, setMessage] = useState('');
+  const [type, setType] = useState('in');
+
+  const submitHandler = async (e) => {
+    e.preventDefault();
+    try {
+      if (type === 'in') {
+        let user = await signInUser(email, password);
+        setUser(user);
+      } else {
+        let user = await signUpUser(email, password);
+        setUser(user);
+      }
+    } catch {
+      setMessage('something got caught');
+    }
+  };
+
+  return (
+    <div>
```

Comment:
Use fragments to return multiple nodes instead of rendering a dom node just to hold them (unnecessary div)